config/settings.py

```diff
@@ -28,9 +33,46 @@
 
 # SECURITY WARNING: don't run with debug turned on in production!
 DEBUG = env("DEBUG")
+DEBUG_TOOLBAR = DEBUG and env("DEBUG_TOOLBAR")
+
+if not DEBUG:
+    sentry_sdk.init(
+        dsn=env('SENTRY_DSN'),
+        integrations=[DjangoIntegration()],
+        traces_sample_rate=1.0,
+        # If you wish to associate users to errors (assuming you are using
+        # django.contrib.auth) you may enable sending PII data.
+        send_default_pii=True
+    )
 
 ALLOWED_HOSTS = []
 
+# START_FEATURE debug_toolbar
+INTERNAL_IPS = ['127.0.0.1']
+# END_FEATURE debug_toolbar
+
+# START_FEATURE django_react
+WEBPACK_LOADER_HOTLOAD = env('WEBPACK_LOADER_HOTLOAD')
+if WEBPACK_LOADER_HOTLOAD:
+    WEBPACK_LOADER = {
+        'DEFAULT': {
+            'LOADER_CLASS': "config.webpack_loader.DynamicWebpackLoader"
+        }
+    }
+else:
+    WEBPACK_LOADER = {
+        'DEFAULT': {
+            'CACHE': not DEBUG,
+            'BUNDLE_DIR_NAME': 'webpack_bundles/',  # must end with slash
+            'STATS_FILE': os.path.join(BASE_DIR, 'webpack-stats.json'),
+            'POLL_INTERVAL': 0.1,
+            'TIMEOUT': None,
+            'IGNORE': [r'.+\.hot-update.js', r'.+\.map'],
+            'LOADER_CLASS': 'webpack_loader.loader.WebpackLoader',
+        }
+    }
```

Comment:
:question: Is this else clause necessary? If so, could you document (comments) what is customized vs what is provided by default? I don't think we configure this on any of our other apps.